quodlibet/quodlibet/images/hicolor/Makefile

```diff
@@ -0,0 +1,82 @@
+QL_SOURCE=scalable/apps/quodlibet.svg
+QL_DEST=quodlibet.png
+
+QL_SYM_SOURCE=scalable/apps/quodlibet-symbolic.svg
+QL_SYM_DEST=quodlibet-symbolic.png
+
+QL_MISSING_COVER_SOURCE=scalable/status/quodlibet-missing-cover.svg
+QL_MISSING_COVER_DEST=quodlibet-missing-cover.png
+
+EF_SOURCE=scalable/apps/exfalso.svg
+EF_DEST=exfalso.png
+
+EF_SYM_SOURCE=scalable/apps/exfalso-symbolic.svg
+EF_SYM_DEST=exfalso-symbolic.png
+
+APPS_DIR=16x16/apps 24x24/apps 32x32/apps 48x48/apps 64x64/apps
+STATUS_DIR=16x16/status 24x24/status 32x32/status 48x48/status 64x64/status
+
+all: mkdestdir png16 png24 png32 png48 png64
```

Comment:
The formatting went off, because copy and pasted it. I tried it with the "&&" with a "{list;}". Seems to work, but is that the right way to have the png files commands executed only if the mkdir command is successful. Or does it not matter?